filters/include/pcl/filters/model_outlier_removal.h

```diff
@@ -48,20 +48,39 @@ namespace pcl
 {
   /** \brief @b ModelOutlierRemoval filters points in a cloud based on the distance between model and point.
    * \details Iterates through the entire input once, automatically filtering non-finite points and the points outside
-   * the model specified by setSampleConsensusModelPointer() and the threshold specified by setThreholdFunctionPointer().
    * <br><br>
    * Usage example:
    * \code
+   #include <pcl/filters/filter_indices.h>
+
+
    * pcl::ModelCoefficients model_coeff;
    * model_coeff.values.resize(4);
-   * model_coeff.values[0] = 0; model_coeff.values[1] = 0; model_coeff.values[2] = 1.5; model_coeff.values[3] = 0.5;
+   * model_coeff.values[0] = 0; 
+   * model_coeff.values[1] = 0; 
+   * model_coeff.values[2] = 1.5; 
+   * model_coeff.values[3] = 0.5;
+
+   * pcl::SACSegmentation<pcl::PointXYZ> seg; 
+   * pcl::PointIndices::Ptr inliers(new pcl::PointIndices);
    * pcl::ModelOutlierRemoval<pcl::PointXYZ> filter;
```

Comment:
Now you are not using the ModelOutlierRemoval at all 😁  ?

Which its documenting?

Why the use of SACSegmentation?

Reply:
* filter.setFilterLimitsNegative (false); ->  * filter.setNegative (false);

Should be sufficient for that part?

And the removal of the two functions mentioned is fine.

Reply:
Cool, I'll make these changes